src/css/dark-theme.css

```diff
@@ -518,11 +518,6 @@ progress[value]::-webkit-progress-bar {
     border-bottom: 1px solid red;
 }
 
-.tab-osd .preview {
-    background: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), url(../images/osd-bg-1.jpg);
-    background-size: cover;
-}
```

Comment:
It will be good if you explain in the PR why you remove this. I suppose the graphic glitches you are seeing come from this?

Reply:
Yes the glitch is removed by removing these lines as they are available here:

https://github.com/betaflight/betaflight-configurator/blob/master/src/css/tabs/osd.css#L486-L508

Reply:
There is no lnear-gradient in this code. Maybe it was a way to make the preview "darker" in the dark theme? Just after that, you have the same but only when active, maybe this was the reason of the square when drag&drop?

Reply:
Just checked again on Ubuntu 20.04 LTS and 22.04 LTS. This part is somehow not an issue anymore so removed now.
The fix in #2913 is still needed to remove the square on drag&drop.